plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/explorer/OpenShiftExplorerLabelProvider.java

```diff
@@ -142,8 +146,6 @@ public StyledString getStyledText(Object element) {
 			case ResourceKind.SERVICE:
 				IService service = (IService) resource;
 				return getStyledText(service);
-			case ResourceKind.TEMPLATE:
```

Comment:
we're removing support for templates since we're not displaying them any more, right? Shouldnt we remove the other types we're not displaying either?